Quests/Buster Quests/co_052103_01.csv

```diff
@@ -4,6 +4,6 @@ linewindow#0,"""Mana collection completed!"""
 11_voice_navi09_war_044#0,"""We've collected enough Mana! With this, you should be able to summon the Buster Pile!"""
 11_voice_navi09_war_045#0,"""<%me>!<br>It seems the enemy's assault has begun! Let's concentrate on protecting the Buster Tower!"""
 text00#0,"""About the Defense Phase"""
-text01#0,"""<c 00ffff>Buster Piles<c> can destroy the devil barriers<br>created from the Devil Castle.<br><br>They are automatically summoned during the<br>Buster Phase and head towards the Devil Castle.<br><br><c ff4f3f>When all of the Buster Piles that are summoned<br>hit and destroy the Devil Barriers, or if they're destroyed,<br>the Buster Phase ends.<c>"""
+text01#0,"""<c 00ffff>Buster Piles<c> can destroy the Devil Barriers<br>created from the Devil Castle.<br><br>They are automatically summoned during the<br>Buster Phase and head towards the Devil Castle.<br><br><c ff4f3f>When all of the Buster Piles that are summoned<br>hit and destroy the Devil Barriers, or if they're destroyed,<br>the Buster Phase ends.<c>"""
```

Comment:
I thought this line should be describing about Defense Phase.

Reply:
Ahhhh, this is what you meant. Good catch! I'll fix it, thanks.